File: skip-the-dishes-query-event-handler/src/main/java/br/com/skip/the/dishes/query/event/handler/commons/EventStoreApiService.java
package br.com.skip.the.dishes.query.event.handler.commons;

import feign.Headers;
import feign.Param;
import feign.RequestLine;

@Headers("Accept: application/json")
public interface EventStoreApiService {

    @RequestLine("POST /projection/{name}/command/enable")
    void enableProjection(@Param("name") String name);

}

File: skip-the-dishes-query-event-handler/src/main/kotlin/br/com/skip/the/dishes/query/event/handler/commons/EventStoreApiService.kt
package br.com.skip.the.dishes.query.event.handler.commons

import feign.Headers
import feign.Param
import feign.RequestLine

@Headers("Accept: application/json")
interface EventStoreApiService {

    @RequestLine("POST /projection/{name}/command/enable")
    fun enableProjection(@Param("name") name: String)

}
